db/seeds.rb

```diff
@@ -6,3 +6,16 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+if ENV['USE_DEMO_DATA'] || Settings.use_demo_data || Rails.env.test?
```

Comment:
- It's ok to have it in the playground, but that shouldn't imply any modifications to the MessagesController. The playground should remain isolated enough to be easily removed in the future without having to change anything else.

- I like the idea of using sidekiq. But add a rake task to allow any user to create a batch of tests on demand without having to setup the daily job. It could just trigger the same sidekiq job. Accept the number of tests to be generated as a parameter.

- I don't have any preferences regarding if you move it to an individual file or to the task setup.